README.md

```diff
@@ -1,14 +1,2112 @@
+
+```
+* creating lib/app_web/controllers/item_controller.ex
+* creating lib/app_web/templates/item/edit.html.eex
+* creating lib/app_web/templates/item/form.html.eex
+* creating lib/app_web/templates/item/index.html.eex
+* creating lib/app_web/templates/item/new.html.eex
+* creating lib/app_web/templates/item/show.html.eex
+* creating lib/app_web/views/item_view.ex
+* creating test/app_web/controllers/item_controller_test.exs
+* creating lib/app/todo/item.ex
+* creating priv/repo/migrations/20200521145424_create_items.exs
+* creating lib/app/todo.ex
+* injecting lib/app/todo.ex
+* creating test/app/todo_test.exs
+* injecting test/app/todo_test.exs
+
+Add the resource to your browser scope in lib/app_web/router.ex:
+
+    resources "/items", ItemController
+
+
+Remember to update your repository by running migrations:
+
+    $ mix ecto.migrate
+```
+
+That created a _bunch_ of files!
+Some of which we don't strictly _need_. <br />
+We could _manually_ create _only_ the files we _need_,
+but this is the "official" way of creating a CRUD App in Phoenix,
+so we are using it for speed.
+
+
+> **Note**: Phoenix
+[Contexts](https://hexdocs.pm/phoenix/contexts.html)
+denoted in this example as `Todo`,
+are "_dedicated modules that expose and group related functionality_."
+We feel they _unnecessarily complicate_ basic Phoenix Apps
+with layers of "interface" and we _really_ wish we could
+[avoid](https://github.com/phoenixframework/phoenix/issues/3832) them.
+But given that they are baked into the generators,
+and the _creator_ of the framework
+[_likes_](https://youtu.be/tMO28ar0lW8?t=376) them,
+we have a choice: either get on board with Contexts
+or _manually_ create all the files in our Phoenix projects.
+Generators are a _much_ faster way to build!
+_Embrace_ them,
+even if you end up having to `delete` a few
+unused files along the way!
+
+We are _not_ going to explain each of these files
+at this stage in the tutorial because
+it's _easier_ to understand the files
+as you are _building_ the App!
+The purpose of each file will become clear
+as you progress through editing them.
+
+
+
+<br />
+
+### 2.1 Add the `/items` Resources to `router.ex`
+
+Follow the instructions noted by the generator to
+add the `resources "/items", ItemController` to the `router.ex`.
+
+Open the `lib/app_web/router.ex` file
+and locate the line: `scope "/", AppWeb do` <br />
+Add the line to the end of the block.
+e.g:
+
+```elixir
+scope "/", AppWeb do
+  pipe_through :browser
+
+  get "/", PageController, :index
+  resources "/items", ItemController # this is the new line
+end
+```
+
+Your `router.ex` file should look like this:
+[`router.ex#L20`](https://github.com/dwyl/phoenix-todo-list-tutorial/blob/f66184b58b7dd1ef593680e7a1a446247909cae7/lib/app_web/router.ex#L20)
+
+<br />
+
+### 2.2 _Run_ The App!
+
+At this point we _already_ have a functional Todo List
+(_if we were willing to use the default Phoenix UI_). <br />
+Try running the app on your `localhost`:
+
```

Comment:
Run the generated migrations with `mix ecto.migrate` then the server with:

Reply:
Another great catch! 🙌